specification.markdown

```diff
@@ -403,6 +403,8 @@ The `VariableParentOccurrence` part is interpreted as an ABNF Variable Repetitio
 
 If the path contains an `EBMLPathAtomRecursive` part, the `EBML Element` can occur within itself recursively (see the [recursive attribute](#recursive)).
 
+As an example, the `path` for `EBMLMaxIDLength`, according to its [definition](#ebmlmaxidlength-element), translates to "1*1(\EBML\EBMLMaxIDLength)". Meaning it's found insided the `EBML` at least once and at most once.
+
```

Comment:
Meaning it's found inside the `EBML` exactly once.

Reply:
Yes, it's explaining the 1 and the 1... Which is not a good example...